### PR TITLE
Migrate search to Compose

### PR DESCRIPTION
**Pull request by chrisbanes**
Screenshot_1602849206

### CODE REVIEW

File: ui-search/src/main/java/app/tivi/home/search/Search.kt [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    state: SearchViewState,
+    actioner: (SearchAction) -> Unit
+) {
+    Box(Modifier.fillMaxSize()) {
+        val searchBarHeight = remember { mutableStateOf(0) }
+
+        SearchList(
+            results = state.searchResults,
+            contentPadding = PaddingValues(
+                top = with(DensityAmbient.current) { searchBarHeight.value.toDp() }
```
Comment (nickbutcher) [inline ui-search/src/main/java/app/tivi/home/search/Search.kt:78]: Couldn't this use the `Insets.toPaddingValues` fun?

Comment (chrisbanes) [reply ui-search/src/main/java/app/tivi/home/search/Search.kt:78]: Nah, it needs to be the status bar height + search bar.

File: ui-search/src/main/java/app/tivi/home/search/SearchFragment.kt [hunk 2]
```diff
@@ -18,97 +18,71 @@ package app.tivi.home.search
...
+        layoutParams = ViewGroup.LayoutParams(MATCH_PARENT, MATCH_PARENT)
 
-    override fun onViewCreated(binding: FragmentSearchBinding, savedInstanceState: Bundle?) {
-        binding.searchRecyclerview.apply {
-            setController(controller)
-            addOnScrollListener(HideImeOnScrollListener())
-        }
+        setContent {
+            MdcTheme {
+                LogCompositions("MdcTheme")
```
Comment (nickbutcher) [inline ui-search/src/main/java/app/tivi/home/search/SearchFragment.kt:67]: Do you want to keep these logs?

**Review by nickbutcher — APPROVED** [review]

File: ui-search/src/main/java/app/tivi/home/search/Search.kt (hunk 1 above)
Comment (nickbutcher) [reply ui-search/src/main/java/app/tivi/home/search/Search.kt:78]: Ahhh yeah.

File: ui-search/src/main/java/app/tivi/home/search/SearchFragment.kt (hunk 2 above)
Comment (chrisbanes) [reply ui-search/src/main/java/app/tivi/home/search/SearchFragment.kt:67]: For now yeah, I'll go through and take a pass through soon and remove them all

**Pull request merged**